GUI/Main.cs

```diff
@@ -21,10 +21,11 @@ public enum GUIModFilter
         Compatible = 0,
         Installed = 1,
         InstalledUpdateAvailable = 2,
-        NewInRepository = 3,
-        NotInstalled = 4,
-        Incompatible = 5,
-        All = 6
+        Cached = 3,
```

Comment:
I know that we save our filter in as `ActiveFilter` in `GUIConfig.xml`, which meas that re-ordering these means that users may mysteriously have their default filter change if they're coming from an older version.

Is there any chance we can add `Cached` to the end of the `enum` rather than in the middle?